main.cpp

```diff
@@ -181,14 +195,25 @@ bool MovieTheater::chargeCashForFoodAndSnacks(bool payingWithCash)
     return ! payingWithCash;
 }
 
+void MovieTheater::printTickets(int howMany)
+{
+    for (int i = 0; i < howMany; ++i)
+    {
+        std::cout << "Printing ticket #" << i + 1 << ". Tickets avaialble: " << --ticketsAvailable << std::endl;
+        
+    }
+}
+
 // UDT 3
 struct Dog 
 {
     int energyLevel;
     int numLegs = 4;
     std::string name;
     int age = 4;
-    int numEyes = 2;
+    int numEyes = 2;   
+    std::string toys[4] = {"Tennis Ball", "Rattlesnake", "Plushie", "Basketball"};
```

Comment:
Arrays have not been covered yet in the course
- [ ] revise: remove this array.